catalog/app/containers/Bucket/Listing.js

```diff
@@ -119,6 +119,10 @@ const useStatsStyles = M.makeStyles((t) => ({
     lineHeight: '15px',
     marginLeft: -4,
   },
+  clearIconBtn: {
+    position: 'absolute',
+    right: -28,
```

Comment:
i think we should use `t.spacing` mostly for spacing btw elements rather than for element dimensions and stuff. this offset value has been adjusted by hand to fit properly. anyways, this code has been removed in a subsequent PR